modules/core/src/test/scala/org/tessellation/http/routes/TrustRoutesSuite.scala

```diff
@@ -35,6 +36,7 @@ object TrustRoutesSuite extends HttpSuite {
           ts = TrustStorage.make[IO](trust)
           gossip = new Gossip[IO] {
             override def spread[A <: AnyRef: TypeTag](rumorContent: A): IO[Unit] = IO.unit
+            override def spreadCommon[A <: AnyRef: universe.TypeTag](rumorContent: A): IO[Unit] = IO.unit
```

Comment:
I think you can remove `universe.`, you have `TypeTag` imported directly

Reply:
True, idk why I used it in the first place 🤦